src/sentry/api/serializers/models/groupsearchview.py

```diff
@@ -0,0 +1,16 @@
+from sentry.api.serializers import Serializer, register
+from sentry.models.groupsearchview import GroupSearchView
+
+
+@register(GroupSearchView)
+class GroupSearchViewSerializer(Serializer):
+    def serialize(self, obj, attrs, user):
```

Comment:
Can you add a response type here?

Reply:
![image](https://github.com/getsentry/sentry/assets/55160142/3f8e6b61-4f03-4eb3-823b-276f6c6b29c1)

Looks like mypy does not like that

Reply:
Are you typing it as a TypedDict? I see other serializers that have done that: https://github.com/getsentry/sentry/blob/8d255a2b22c6d0f7e53ed7a8fe94fb080a2060d0/src/sentry/monitors/serializers.py#L36

Reply:
Ah yes I was, I'll look at that one and see if I can make it work